src/Command/AssertBackwardsCompatible.php

```diff
@@ -34,6 +35,8 @@
 
 final class AssertBackwardsCompatible extends Command
```

Comment:
Would be a good idea to support `--generate-baseline` 🤔

Reply:
I'd say it's a separate feature that may or not be added.

Since the introduction of BC-breaks isn't something done that often/lightly, I'd start with manual generation.